### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by Andrius86** [r4v10l1/NullHooks#8]
In my opinion the cheat lack of backtrack it would be really nice if you would add backtrack.

**Comment by r4v10l1** [comment]
Yes, I want to add backtrack in the future (and chams for it too).

**Comment by r4v10l1** [comment]
Fixed in 65203bcf581e9e0de053a81a2c822a51eded0e97
